libp2p/transports/tcptransport.nim

```diff
@@ -239,7 +246,10 @@ method accept*(self: TcpTransport): Future[Connection] =
           # Waits for any one of these futures to complete, indicating that a new connection has been accepted on one of the servers.
           await one(self.acceptFuts)
```

Comment:
Was this comment taken care of? https://github.com/vacp2p/nim-libp2p/pull/1164#discussion_r1705136464